pkg/metadata/host/host.go

```diff
@@ -110,6 +112,16 @@ func getHostTags() *tags {
 		hostTags = append(hostTags, k8sTags...)
 	}
 
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
```

Comment:
Any reason why the context is created here instead of inside docker.GetTags ?
Other GetTags method don't have an arg passed - I thought it would be more consistent.

Reply:
I did that because it is the cleanest way to have the caller specify a timeout but you're right it is going against the pattern of other `GetTags` functions so I can change it.